ArcGISRuntimeSDKQt_QMLSamples/DisplayInformation/CreateSymbolStylesFromWebStyles/CreateSymbolStylesFromWebStyles.qml

```diff
@@ -0,0 +1,201 @@
+                                                                                                                                                                     return;
+
+                                                                                                                                                                     const label = symbolResult.key;
+                                                                                                                                                                     const values = getValuesFromSymbolLabel(label);
+                                                                                                                                                                     const symbol = symbolResult.fetchSymbolResult;
+                                                                                                                                                                     values.forEach((value) => {
+                                                                                                                                                                                        const uniqueValue = ArcGISRuntimeEnvironment.createObject("UniqueValue", {label: label, values: [value], symbol: symbol}, webLayer);
+                                                                                                                                                                                        uniqueValueRenderer.uniqueValues.append(uniqueValue);
+                                                                                                                                                                                    });
+                                                                                                                                                                 });
+                                                                                                                   symbolResult.fetchSymbol();
+                                                                                                               });
+                                                                   });
+
+                    symbolStyleSearchParameters.keys = symbolKeys;
+                    symbolStyle.searchSymbols(symbolStyleSearchParameters);
+                }
+            }
+
+            initialViewpoint: ViewpointCenter {
+                center: Point {
+                    x: -118.44186
+                    y: 34.28301
+                    spatialReference: SpatialReference { wkid: 4326 }
+                }
+                targetScale: 7000
+            }
+        }
+        onMapScaleChanged: {
+            // Set scale symbols to true when we zoom in so the symbols don't take up the entire view
+            webLayer.scaleSymbols = (mapView.mapScale >= 80000);
+        }
```

Comment:
would be better to use property binding instead of assignment if possible. 

for example

```
FeatureLayer {
    id: webLayer
    scaleSymbols: mapView.mapScale >= 8000
    ServiceFeatureTable {
        url: "https://services.arcgis.com/V6ZHFr6zdgNZuVG0/arcgis/rest/services/LA_County_Points_of_Interest/FeatureServer/0"
    }
```